### PR TITLE
New go 1.18

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,245 @@
...
-aos_vis is web socket server which provides vehicle information data using W3C protocol
+aos_vis is the WebSocket server that provides vehicle information data using [W3C protocol](https://www.w3.org/TR/vehicle-information-service/)
+
+## Extendable adapters
 
-https://www.w3.org/TR/vehicle-information-service/
+aos_vis supports several adapaters. Configuration for each adapter could be specified via a configuration file.
 
-### Build
+### Vinadapter plugin
```
Comment (al1img) [inline README.md:12]: Start with small letter: `vinadapter plugin` or `VIN adapter (vinadapter)`

File: README.md [hunk 2]
```diff
@@ -1,54 +1,245 @@
...
+        "VISPath": "Attribute.Vehicle.VehicleIdentification.VIN",
+        "FilePath": "/var/aos/vis/vin"
+    }
+}
 ```
-go build -buildmode=plugin plugins/storageadapter/storageadapter.go
-go build -buildmode=plugin plugins/telemetryemulatoradapter/telemetryemulatoradapter.go
-go build -buildmode=plugin plugins/renesassimulatoradapter/renesassimulatoradapter.go
+
+### Boardmodeladapter
```
Comment (al1img) [inline README.md:27]: ditto

File: README.md [hunk 3]
```diff
@@ -1,54 +1,245 @@
...
+    "Plugin": "boardmodeladapter",
+    "Params": {
+        "VISPath": "Attribute.Aos.BoardModel",
+        "FilePath": "/etc/aos/board_model"
+    }
+}
 ```
 
-Build main program:
+### Subjectsadapter
```
Comment (al1img) [inline README.md:42]: ditto

File: README.md [hunk 4]
```diff
@@ -1,54 +1,245 @@
...
+    "Plugin": "boardmodeladapter",
+    "Params": {
+        "VISPath": "Attribute.Aos.Subjects",
+        "FilePath": "/var/aos/subjects"
+    }
+}
 ```
-go build
+
+### Renesassimulatoradapter
```
Comment (al1img) [inline README.md:57]: ditto

File: README.md [hunk 5]
```diff
@@ -1,54 +1,245 @@
...
+
+Performs conversation from Renesas simulator data to VIS protocal
```
Comment (al1img) [inline README.md:59]: `conversion` or better `Converts Renesas simulator data to VIS protocol`

File: README.md [hunk 6]
```diff
@@ -1,54 +1,245 @@
...
+            "geometry.coordinates.Longitude": "Signal.Cabin.Infotainment.Navigation.CurrentLocation.Longitude",
+            "geometry.coordinates.Altitude": "Signal.Cabin.Infotainment.Navigation.CurrentLocation.Altitude",
+            "geometry.coordinates.HorizAccu": "Signal.Cabin.Infotainment.Navigation.CurrentLocation.Accuracy"
+        }
+    }
+}
 ```
 
-### Configure
+### Telemetryemulatoradapter
```
Comment (al1img) [inline README.md:77]: ditto

**Review by al1img — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

**Pull request merged**